perch/main.py

```diff
@@ -42,6 +43,19 @@
 }
 
 
+def check_version(ctx):
+    pypi_url = 'https://pypi.org/pypi/perch/json'
+    res = requests.get(pypi_url)
+    pypi_info = res.json()
+    latest_version = pypi_info['info']['version']
+    if __version__ != latest_version:
+        click.echo(
+            message='Your perch client is out of date! Please upgrade your client using: pip install perch -U',
```

Comment:
I might suggest putting the command on its own line:
```python
message='Your perch client is out of date! Please upgrade your client using:\n\n    pip install -U perch\n'
```

![screen shot 2018-06-04 at 17 42 53](https://user-images.githubusercontent.com/33840/40943386-da93cc40-681e-11e8-938f-b21855aca42f.png)

vs

![screen shot 2018-06-04 at 17 43 12](https://user-images.githubusercontent.com/33840/40943426-0c87a884-681f-11e8-92a8-24bb1db19bbe.png)